src/theme/formStyles.js

```diff
@@ -164,6 +162,57 @@ const validationStyle = {
 	color: crimson.default,
 };
 
+const selectStyle = {
+	placeholder: (provided) => ({
+		...provided,
+		...longPrimer,
+	}),
+	option: (provided) => ({
+		...provided,
+		borderRadius: 6,
+	}),
+	valueContainer: (provided) => ({
+		...provided,
+		...longPrimer,
+	}),
+	menu: (provided) => ({
+		...provided,
+		...longPrimer,
+		borderRadius: '8px',
+		marginTop: '4px',
+		padding: '16px',
+	}),
+	control: (provided, state) => ({
+		...provided,
+		borderRadius: 6,
+		borderColor: state.isDisabled ? tuna.lighter2 : provided.primary,
+	}),
+	multiValueLabel: (provided) => ({
+		...provided,
+		...longPrimer,
+	}),
+};
+
+const errorSelectStyle = {
+	...selectStyle,
+	control: (provided) => ({
+		...provided,
+		borderRadius: 6,
+		borderColor: crimson.default,
+		'&:hover': {
+			borderColor: crimson.default,
+		},
+	}),
+	option: (provided, state) => ({
+		...provided,
+		borderRadius: 6,
+		backgroundColor: state.isSelected ? primary.default : coconut.default,
+		'&:hover': {
+			backgroundColor: state.isSelected ? primary.default : primary.lighter2,
```

Comment:
use a border instead of a different background for the hover of an unselected option

Reply:
Made the onHover background color lighter to increase contrast